pkg/operator/workloads/api_workload.go

```diff
@@ -635,7 +635,7 @@ func APIsBaseURL() (string, error) {
 	if len(service.Status.LoadBalancer.Ingress) == 0 {
 		return "", ErrorLoadBalancerInitializing()
 	}
-	return "https://" + service.Status.LoadBalancer.Ingress[0].Hostname, nil
+	return "http://" + service.Status.LoadBalancer.Ingress[0].Hostname, nil
```

Comment:
We should tell you somewhere in the docs that you can use https (and that it will use a cert that we generated on install), can you add that? Omer should have an opinion on where to add it in the docs.